TBDataPreparation/202108_SPS/scripts/PhysicsConverter.C

```diff
@@ -106,24 +102,17 @@ void PhysicsConverter(const string run){
     //
     ev->calibrate(sipmCalibration, evout);
     ev->calibratePMT(pmtCalibration, evout);
-    evout->CompSPMTene();
-    evout->CompCPMTene();
     //std::cout<<ev->EventID<<" "<<ev->totSiPMPheS<<std::endl;
     //Write event in ftree
     //
     ftree->Fill();
-    //Reset totSiPMPheC and totSiPMPheS to 0
-    //
-    evout->totSiPMCene = 0;
-    evout->totSiPMSene = 0;
```

Comment:
Commented below. Maybe we should not including the energy scale setting to zero inside the calibrating functions. In it is a bit counterintuitive and the two things should be independent in my view.